src/Application/Generator/DateTimeGenerator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the test generator project.
+ *
+ * (c) Alexander Schranz (https://github.com/alexander-schranz/test-generator)
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Schranz\TestGenerator\Application\Generator;
+
+/**
+ * @internal
+ */
+class DateTimeGenerator
```

Comment:
needed to have fixed datetimes for the testcases

Reply:
it should not create 100% occur code would jsut create a datetime with `1970-01-01` always

Reply:
second datetime with `1970-01-02` like its done for the integers